plugins/woocommerce/legacy/js/flexslider/jquery.flexslider.js

```diff
@@ -1141,6 +1163,16 @@
     focused = false;
   }).on( 'focus', function ( e ) {
     focused = true;
+  }).on( 'load', function () {
+	  $('.woocommerce-product-gallery__wrapper .flexslider-deferred').each(function () {
+		  $(this)
+			  .prop({
+				  src: $(this).attr('data-src'),
+				  srcset: $(this).attr('data-srcset')
+			  })
+			  .removeAttr('data-src data-srcset loading')
+			  .removeClass('flex-deferred');
```

Comment:
wrong class -> update to "flexslider-deferred" (anyhow it is a cosmetic issue and doesn't affect how image defer works)